layouts/_default/baseof.html

```diff
@@ -33,7 +33,7 @@
     {{ .Hugo.Generator }}
   </head>
 
-  <body>
+  <body {{ if (.Site.Params.invertedColors | default false) }}class="inverted-colors"{{ end }}>
```

Comment:
I think you don't need the `default false` here. If I'm not mistaken, undeclared parameters are always zero-value, so, also false.

Reply:
thx for pointing it out, changed 👌